modules/core/src/main/scala/dev/profunktor/redis4cats/connection/RedisClient.scala

```diff
@@ -31,9 +31,8 @@ object RedisClient {
   private[redis4cats] def acquireAndRelease[F[_]: Concurrent: ContextShift: Log](
       uri: => RedisURI,
       opts: ClientOptions,
-      config: Redis4CatsConfig,
-      redisEc: RedisEc
-  ): (F[RedisClient], RedisClient => F[Unit]) = {
+      config: Redis4CatsConfig
+  )(implicit redisEc: RedisEc[F]): (F[RedisClient], RedisClient => F[Unit]) = {
```

Comment:
Sorry for nitpicking but can we add `RedisEc` as a context bound instead? So it's aligned with the other constraints. You probably need to add a "summoner" in the companion object.